README.md

```diff
@@ -28,12 +28,12 @@ echo WP_DEBUG_DISPLAY;
 
 Just run `make` to do everything
 
-We use PHPUnit 5 because it has compatibility for PHP 5.6
+We use PHPUnit 7 because it has compatibility for PHP 7.1
```

Comment:
```suggestion
We use PHPUnit
```

Reply:
Shouldn't we explain/warn developers about why we not using the latest versions of PHPUnit?